### PR TITLE
Design refinement - RichTextEditor

### PR DESCRIPTION
**Pull request by mariush2**
- [AB#432423](https://dev.azure.com/Equinor/4319082b-29ba-4111-a20b-c04f55c7e7c6/_workitems/edit/432423)
- [AB#460322](https://dev.azure.com/Equinor/4319082b-29ba-4111-a20b-c04f55c7e7c6/_workitems/edit/460322)

### CODE REVIEW

File: src/components/Inputs/RichTextEditor/RichTextEditor.stories.tsx [hunk 1]
```diff
@@ -15,9 +15,14 @@ export default {
       options: [...DEFAULT_FEATURES],
     },
     onChange: { action: 'Fired onChange fn' },
+    onImageUpload: { action: 'Fired onImageUpload fn' },
   },
   args: {
     features: [...DEFAULT_FEATURES],
+    onImageUpload: async (file: File) => {
+      console.log(file.name);
```
Comment (aslakihle) [inline src/components/Inputs/RichTextEditor/RichTextEditor.stories.tsx:23]: Not sure if this console log is intentional 🤔 😂

File: src/components/Inputs/RichTextEditor/custom-extensions/ExtendedImage.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      new Plugin({
+        props: {
+          handleDOMEvents: {
+            drop(view, event) {
+              const hasFiles =
+                event.dataTransfer &&
+                event.dataTransfer.files &&
+                event.dataTransfer.files.length;
+
+              console.log(this);
```
Comment (aslakihle) [inline src/components/Inputs/RichTextEditor/custom-extensions/ExtendedImage.ts:35]: console.log here aswell c:

**Review by aslakihle — CHANGES_REQUESTED** [review]

File: src/components/Inputs/RichTextEditor/RichTextEditor.stories.tsx (hunk 1 above)
Comment (mariush2) [reply src/components/Inputs/RichTextEditor/RichTextEditor.stories.tsx:23]: Just for the storybook story 😄 Thought id be nice to see what you 'uploaded' in the console

File: src/components/Inputs/RichTextEditor/custom-extensions/ExtendedImage.ts (hunk 2 above)
Comment (mariush2) [reply src/components/Inputs/RichTextEditor/custom-extensions/ExtendedImage.ts:35]: This one however, isn't intentional 👓

**Review by aslakihle — APPROVED** [review]

**Pull request merged**